bin/cheribsdtest/cheribsdtest_vm.c

```diff
@@ -2623,6 +2623,40 @@ CHERIBSDTEST(cheri_revoke_cow_mapping,
 
 	cheribsdtest_success();
 }
+
+CHERIBSDTEST(cheri_revoke_shm_anon_hoard_unmapped,
+    "Capability is revoked within an unmapped shm object",
+    .ct_xfail_reason = "shm segments aren't revoked")
```

Comment:
"unmapped parts of shm segments", perhaps?  Were it mapped, we'd revoke at least the mapped pages, even if not the whole object, yes?